api/admin/controllers/user.js

```diff
@@ -192,7 +197,7 @@ module.exports = wrapHandlers({
       user,
     } = req;
 
-    const { email, inviteLink: link } = await OrganizationService.resendInvite(
+    const { email, inviteLink: link, origin } = await OrganizationService.resendInvite(
```

Comment:
ok so some of them don't get org.name. If we did pass it on to the mailer, I think it would be lovely to let the user know which org they were invited to. I know it's only for the first org they join, but it would be nice if they could recognize their team/project in the invite email, and most folks are probably in just one org.

Reply:
I was pondering this too. The current dilemma is the admin UI is setup to resend invites to the user for the entire platform and it's not org specific. Also, we only need to resend an invite code if the user's origin is cloud.gov. I think we should examine the JTBD for an admin user in this scenario.